File: 0217-contains-duplicate/0217-contains-duplicate.py
"""
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        occurred = {}
        for n in nums:
            if occurred.get(n) != None:
                return True
            
            occurred[n] = True
        
        return False
"""

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return len(nums) != len(set(nums))

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    occ = new Map()
    for (num of nums) {
        if (occ.has(num)) {
            return true
        }
        occ.set(num, true)
    }
    return false
};